tests/python_callback_test.py

```diff
@@ -12,43 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from collections.abc import Sequence
 import functools
 import textwrap
-from typing import Any, Callable
 import unittest
 
-from absl.testing import absltest
+from absl.testing import absltest, parameterized
```

Comment:
Nit: google style guide suggests we split these imports across multiple lines

Reply:
Will fix. Meanwhile I moved this to an internal CL, and will close this PR.